app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt

```diff
@@ -281,8 +282,12 @@ class SearchResultsFragment : Fragment(), CompoundButton.OnCheckedChangeListener
         }
 
         rootView.searchText.addTextChangedListener(object : TextWatcher {
-            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) { /*Do Nothing*/ }
-            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) { /*Do Nothing*/ }
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) { /*Do Nothing*/
+            }
+
+            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) { /*Do Nothing*/
+            }
+
```

Comment:
why did you make these changes?

Reply:
@nikit19 I just reformat the code for proper alignment nothing change in it except the search result observer. Is there  any change required?

Reply:
It was correct already

Reply:
@nikit19 so do I need to update the PR with above changes restored as the original one in SearchResultFragment.kt file keeping the search result observer same.

Reply:
Yes